### PR TITLE
Weather Wiki Changes

### PR DESCRIPTION
**Pull request by Amanda-Catherine**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -267,14 +267,32 @@
...
+	'wgAbuseFilterBlockDuration' => array(
+		'default' => 'indefinte',
+		'weatherwiki' => '1 week',
+	),
 	'wgAbuseFilterAnonBlockDuration' => array(
 		'default' => 2592000,
-	),
+		'weatherwiki' => '72 hours',
+	),
+	'wgAbuseFilterRestrictions' => array(
```
Comment (The-Voidwalker) [inline LocalSettings.php]: Unclosed array

File: LocalSettings.php [hunk 2]
```diff
@@ -267,14 +267,32 @@
...
+ -		'default' => array(
+ -			'blockautopromote' => true,
+ -			'block' => true,
+ -			'degroup' => true,
+ -			'rangeblock' => true,
+ -		),
+ -		'weatherwiki' => array(
```
Comment (The-Voidwalker) [inline LocalSettings.php]: Unclosed array

File: LocalSettings.php [hunk 3]
```diff
@@ -5005,6 +5136,25 @@
 				'commentadmin',
 			),
 		),
+		'+weatherwiki' => array(
+ 			'founder' => array(
```
Comment (The-Voidwalker) [inline LocalSettings.php]: Unclosed array

File: LocalSettings.php [hunk 4]
```diff
@@ -4639,6 +4681,95 @@
 				'upload' => true,
 			),
 		),
+			'+weatherwiki' => array(
```
Comment (The-Voidwalker) [inline LocalSettings.php]: Extra indent (tab)

File: LocalSettings.php [hunk 5]
```diff
@@ -267,14 +267,32 @@
...
+ -			'blockautopromote' => true,
+ -			'block' => true,
+ -			'degroup' => true,
+ -			'rangeblock' => true,
+ -		),
+ -		'weatherwiki' => array(
+ -			'blockautopromote' => false,
+ -			'block' => false,
+ -			'degroup' => false,
+ -			'rangeblock' => true,
```
Comment (MacFan4000) [inline LocalSettings.php]: Unclosed arrays

File: LocalSettings.php [hunk 6]
```diff
@@ -5005,6 +5137,25 @@
...
+ 				'founder',
+ 				'ipblock-exempt',
+				'banned',
+ 				'researcher',
+ 				'page-mover',
+ 				'importer',
+ 				'uploader',
+ 				'account-creator',
+ 				'massmessage-sender',
+
```
Comment (MacFan4000) [inline LocalSettings.php]: Add a ), here

**Comment by Amanda-Catherine** [comment]
@MacFan4000 @The-Voidwalker think it’s good to go now (check passed)

**Pull request merged**